### ISSUE TITLE
Add option to place all DLC files in a folder

### ISSUE THREAD

**Issue by Golumpa** [trembon/switch-library-manager#55]
It would be nice to be able to have a toggle whether or not DLC files are put in a DLC folder or left in the base game folder. Some games such as Taiko no Tatsujin Drum 'n' Fun have over 300 DLC's so a way to separate them and the main game files would be useful.

such as:
```
game.nsp
update.nsp
DLC/dlc1.nsp
DLC/dlc2.nsp
DLC/dlc3.nsp
```

![image](https://github.com/user-attachments/assets/6d7a3d63-4abf-470f-a233-0d5c34ab9e39)
![image](https://github.com/user-attachments/assets/186ac664-76dd-4786-a614-1da2694a1024)

**Comment by trembon** [comment]
available in the latest build with the options "dlc_folder" and "updates_folder"

**Comment by SteveMyers75** [comment]
> available in the latest build with the options "dlc_folder" and "updates_folder"

I'm really excited about this new function, but I'm not able to get this to work just yet. 

Using the Windows version of v17. on Windows 10 Pro

I put a few Base titles with their matching Updates and DLC in a folder, scanned it then ran Organize/Begin Organization.
Nothing happens  (No folders are created)

I can see the "**Create Updates folders**" and "**Create DLC folders**" settings in the **Organize** tab, so I know the new v17 compiled correctly.

My settings look like:
```
"organize_options": {
    "create_folder_per_game": false,
    "dlc_folder": "N:\\Switch\\Games\\DLC",
    "updates_folder": "N:\\Switch\\Games\\Updates",
    "rename_files": true,
    "delete_empty_folders": false,
    "delete_old_update_files": true,
    "folder_name_template": "{TITLE_NAME}",
    "switch_safe_file_names": true,
    "file_name_template": "{TITLE_NAME} ({DLC_NAME})[{TITLE_ID}][v{VERSION}]",
    "process_when_missing_base_game": false
  },
```
What am I doing wrong?

**Comment by trembon** [comment]
the idea behind this feature is that when you have the create_folder_per_game set to true, and have the value dlc_folder set for example to "DLCs", it will create a folder inside the games folder named "DLCs" and place all the DLCs in there

so what doesnt with your configuration is that the dlc folder is not an absolut path and needs the create_folder_per_game to be set

**Comment by trembon** [comment]
released with 1.8.0

**Comment by NikoMuffin** [comment]
> the idea behind this feature is that when you have the create_folder_per_game set to true, and have the value dlc_folder set for example to "DLCs", it will create a folder inside the games folder named "DLCs" and place all the DLCs in there
> 
> so what doesnt with your configuration is that the dlc folder is not an absolut path and needs the create_folder_per_game to be set


Can confirm this works.
